generators/PleadingKitten/index.ts

```diff
@@ -0,0 +1,23 @@
+import { Generator } from '../../types/GeneratorTypes';
+import { SettingType } from '../../types/SettingTypes';
+import { generate } from './generate';
+import { PleadingKittenSettings } from './types';
+
+export const pleadingKittenGenerator: Generator<PleadingKittenSettings> = {
+  name: 'Pleading kitten',
```

Comment:
Since there's already a "pleading boykisser" generator, your other suggestion of "attention-starved kitten" might be better (I appreciate renaming the entire generator is the most annoying thing to do)